### ISSUE TITLE
Bug in mobile mode on Chrome desktop when using tab

### ISSUE THREAD

**Issue by Makio64** [Mwni/pixi-text-input#26]
When you use tab in mobile mode on Chrome desktop then you cant focus anymore by clicking on the field.

**Comment by Mwni** [comment]
Good catch! But is it actually possible to do that on a real mobile device? Wont fix otherwise, because no end user will do that on desktop.

**Comment by Nep-Toodles** [comment]
yep it is in mobile too
